modules/graphman-operation-renew.js

```diff
@@ -73,16 +83,19 @@ function renewEntities(gateway, entities, type) {
         buildQueryForSoapServiceEntities(entities, type, typeObj, queryInfo);
     } else if (type === 'FipUser' || type === 'FipGroup') {
```

Comment:
do we have to include FederatedUser and FederatedGroup also ?

Reply:
We will do that separately.